db/views/submissions_by_date_v02.sql

```diff
@@ -0,0 +1,16 @@
+SELECT
```

Comment:
Think we might be missing the migration that actually applies this view?

Reply:
yeah looking into this now - scenic being weird its not creating the versioned files properly on its own

Reply:
potentially the view was renamed and that's messing up scenic working as expected - I can just manually create the correct migrations so going to do that instead